lib/framework/procbot-types.d.ts

```diff
@@ -40,3 +40,8 @@ export interface ConfigurationLocation {
 	 */
 	location: string | any;
 }
+
+declare const enum ProcBotErrorCode {
```

Comment:
You're declaring an existing enumerated group of `ProcBotErrorCode`... but I can't find it? Where's it declared? Should these actually be specific constant values?

Reply:
@hedss hopefully what I've done answers this part of the review, but I'm honestly unsure.